apps/nextjs-website/src/app/profile/chatbot-history/page.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { useTranslations } from 'next-intl';
+import { Stack, Typography } from '@mui/material';
+import React from 'react';
+import ChatbotHistoryLayout from '@/components/organisms/ChatbotHistoryLayout/ChatbotHistoryLayout';
+import { useChatbot } from '@/helpers/chatbot.helper';
+import { useUser } from '@/helpers/user.helper';
+
+const ChatbotHistory = () => {
+  const t = useTranslations('profile.chatbot');
+  const { user, loading } = useUser();
+  const { paginatedSessions, getSessions } = useChatbot(true);
+
+  if (loading || !user || !paginatedSessions) {
+    return <div>Loading...</div>; // Maybe use skeletons here?
+  }
+
+  return (
+    <>
+      <Stack
+        sx={{
+          padding: { xs: '40px 24px', md: '80px 40px' },
+          width: '100%',
+          maxWidth: '694px',
+        }}
+      >
+        <Typography variant='h4' sx={{ marginBottom: '40px' }}>
+          {t('title')}
```

Comment:
Applay this suggestion only if previous one is accepted

```suggestion
          {t('profile.chatbot.title')}
```